app/tests/spec/views/connect_another_device.js

```diff
@@ -306,130 +306,27 @@ define(function (require, exports, module) {
       it('returns `false` if user is signed in', () => {
 
         sinon.stub(user, 'isSignedInAccount', () => true);
-        sinon.stub(view, '_hasWebChannelSupport', () => true);
+        sinon.stub(view, '_webChannelSupportExists', () => true);
 
 
         assert.isFalse(view._canSignIn());
       });
 
       it('returns `false` if no web channel support', () => {
         sinon.stub(user, 'isSignedInAccount', () => false);
-        sinon.stub(view, '_hasWebChannelSupport', () => false);
+        sinon.stub(view, '_webChannelSupportExists', () => false);
 
         assert.isFalse(view._canSignIn());
       });
 
       it('returns `true` if not signed in, has web channel support', () => {
         sinon.stub(user, 'isSignedInAccount', () => false);
-        sinon.stub(view, '_hasWebChannelSupport', () => true);
+        sinon.stub(view, '_webChannelSupportExists', () => true);
 
         assert.isTrue(view._canSignIn());
       });
     });
 
-    describe('_hasWebChannelSupport', () => {
```

Comment:
These tests will need to be migrated to [app/tests/spec/lib/user-agent.js](https://github.com/mozilla/fxa-content-server/blob/master/app/tests/spec/lib/user-agent.js) to ensure we don't accidentally introduce regressions in the future.